docs/conf.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-from __future__ import absolute_import, print_function, unicode_literals
```

Comment:
Here you delete `print_function`, too.. it makes it a bit hard to review if this is also deleted, because it requires to look through subsequent lines that tests don't touch

Reply:
I'll add it back.

Reply:
Still awaiting :)

Reply:
I think Github shows the diff in a strange way. It's indeed there, that's at least what I see with "View".